Gemfile

```diff
@@ -82,7 +82,7 @@ gem "rsolr", ">= 1.0"
 
 gem "aasm"
 gem "arabic-letter-connector"
-gem "browse-everything", github: "jrgriffiniii/browse-everything", branch: "figgy-issues-3587-jrgriffiniii"
+gem "browse-everything", github: "samvera/browse-everything", branch: "2.x-stable"
```

Comment:
Is there a released version?

Reply:
Not yet. Making an issue to move to released branch once it's available: https://github.com/pulibrary/figgy/issues/3805